sale_product_multi_add/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
     "category": "Sale Management",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```

Comment:
@juanpgarza The good flow for migration is to reset major/minor/patch to : 1.0.0.